### PR TITLE
Laravel Pail module file delete

### PR DESCRIPTION
**Pull request by MaxRio87**

### CODE REVIEW

File: gadgetchains/Laravel/FD/1/chain.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace GadgetChain\Laravel;
+
+class FD1 extends \PHPGGC\GadgetChain\FileDelete
+{
+    public static $version = '*';
+    public static $vector = '__destruct';
+    public static $author = 'Maxime Rinaudo (www.fenrisk.com)';
+    public static $information = 'Deletes an arbitrary file in Pail plugin';
```
Comment (jvoisin) [inline gadgetchains/Laravel/FD/1/chain.php:10]:
```suggestion
    public static $information = 'Deletes an arbitrary file in Laravel's Pail plugin';
```

**Comment by cfreal** [comment]
Hello Maxime, PandHacker, jvoisin,

Thanks for the PR and review.

Please, two more things: declare the properties in the class (`public $file`) and refrain from providing an URL in the author section :)

Best regards,
Charles

**Comment by MaxRio87** [comment]
Thank you for your answer, here is the correction :)

**Comment by cfreal** [comment]
Done! Thanks, Maxime!

**Pull request merged**